source/_components/waterfurnace.markdown

```diff
@@ -0,0 +1,68 @@
+---
+layout: page
+title: "Waterfurnace"
+description: "Instructions on how to integrate Waterfurnace Geothermal System into Home Assistant."
+date: 2018-01-19 17:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: waterfurnace.png
+ha_category: Hub
+ha_release: 0.62
+ha_iot_class: "Cloud Polling"
+---
+
+The `Waterfurnace` component communicates with the Waterfurnace
```

Comment:
- Component names are lower-case: `waterfurnance`
- `the Waterfurnace` -> `the WaterFurnance`